app/src/main/java/com/bucketlist/ui/main/MainFragment.kt

```diff
@@ -29,6 +25,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
-
+    companion object {
+        fun newInstance() = MainFragment()
```

Comment:
I changed the order based on the order classes should follow as mentioned here:
https://kotlinlang.org/docs/coding-conventions.html